server/lib/sightingsController.js

```diff
@@ -0,0 +1,30 @@
+// const mongoose = require('mongoose');
+
+const Sighting = require('../models/Sighting.js');
+
+// Create test sighting
+/* const test = Sighting({
+	animal: 'Poof',
+	location: 'Deck',
+	time: 'Now'
+}); */
+
+// Get all sightings in the database
+exports.getSightings = function (pagination = {}, filter = {}) {
```

Comment:
```suggestion
exports.getSightings = function (searchOpts = {}) {
```

Reply:
All options become a single JSON? So it'd be something like:

searchOpts = {
    pagination: xxxxxxx
    filter: yyyyyyy
    foo: zzzz
}

Reply:
Done